lms/djangoapps/teams/errors.py

```diff
@@ -0,0 +1,11 @@
+"""Errors thrown in the Team API"""
+
+
+class NotEnrolledInCourseForTeam(Exception):
```

Comment:
I'd introduce a base class for all the errors to subclass from. Look at ``user_api/errors.py`` for an example. I'd also introduce a TeamAPIInternalError like we did for the User API, so that unexpected errors can be caught and handled cleanly.